reference/api/bindings_api.md

```diff
@@ -80,6 +80,73 @@ def incoming():
     return "Kafka Event Processed!"
 ```
 
+### Binding subscription
+
+On startup Dapr will invoke the following endpoint on user code to discover binding subscriptions:
```

Comment:
This needs rewording: Bindings are discovered from component yamls. Dapr calls this endpoint on startup to ensure that app indeed is handling it, if the app doesnt have the endpoint, dapr ignores it.